packages/did-provider/src/utils.ts

```diff
@@ -49,10 +49,6 @@ export function hexToU8A(s: string): Uint8Array {
   return u8a.fromString(s, B16)
 }
 
-export function u8aToHex(b: Uint8Array): string {
```

Comment:
are we sure this is not used anywhere else?

Reply:
I did a search at the repository level and couldn't find any other occurrence, and the tests pass so I think it's OK.